.changeset/breezy-horses-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Removed dummy `test` text from appearance layout unit tests, with valid URLs for improved readbility.
```

Comment:
This is an internal change not affecting customers in any way, so let's just add an empty changeset here :)
You can add an empty changeset by running `npm run changeset:empty`.

Reply:
Done!